eng/pipelines/corefx-base.yml

```diff
@@ -188,7 +188,7 @@ jobs:
             - template: /eng/pipelines/helix.yml
               parameters:
                 # send tests to helix only on public builds, official scheduled builds or manual official builds.
-                condition: or(eq(${{ parameters.isOfficialBuild }}, False), notIn(variables['Build.Reason'], 'BatchedCI', 'IndividualCI'))
```

Comment:
Github won't let me comment on line 188 but maybe we could instead of using a condition here, use a ${{ if }} condition so that this step doesn't always show as skipped on official builds and that way the logs are smaller there. 

```
- ${{ if eq(parameters.isOfficialBuild, 'false') }}:
  - template: /eng/pipelines/helix.yml
     ...
```

Then you can remove this condition parameter entirely.

Reply:
I thought about it but I don't think this is necessary. This will only be skipped in official builds and nobody looks at it unless something breaks.

Reply:
Yeah, I agree on that but, it is nice to have less steps showed as skipped the logs would be cleaner. Up to you.